src/views/ForgotPasswordPageInput/index.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from "react";
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import {Link, Route} from 'react-router-dom';
+import ForgotPasswordPageChange from '../ForgotPasswordPageChange/index';
```

Comment:
Don't need the '/index' in the import, it automatically knows to take the default class from the index file without being explicitly told!

Reply:
Didn't know that, good to know!